src/Domain/CoordinateMaps/EquiangularMap.hpp

```diff
@@ -0,0 +1,97 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class EquiangularMap.
+
+#pragma once
+
+#include <array>
+#include <memory>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Parallel/CharmPupable.hpp"
+
+namespace CoordinateMaps {
+
+/*!
+ * \ingroup CoordinateMapsGroup
+ * \brief Non-linear map from \f$\xi \in [A, B]\rightarrow x \in [a, b]\f$.
+ *
+ * The formula for the mapping is...
+ * \f[
+ * x = \frac{a}{2} (1-\mathrm{tan}(\frac{\pi(2\xi-B-A)}{4(B-A)}) +
```

Comment:
Have you checked what this looks like rendered? I would guess using `align` and then `&` to align by the equal signs is more desirable

Reply:
I think the equal sign might happen to align in this case as is, but it should be changed to align with `&`'s anyway in case x or xi change to anything that isn't 1 character.